Sources/_TestingInternals/include/InstrProfiling.h

```diff
@@ -0,0 +1,88 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+#if !defined(SWT_INSTR_PROFILING_H)
+#define SWT_INSTR_PROFILING_H
+
+#include "Defines.h"
+#include "Includes.h"
+
+SWT_ASSUME_NONNULL_BEGIN
+
+/// @defgroup LLVM Profile Runtime Interface
+/// @{
+///
+/// These functions are provided by the LLVM profile runtime when code is
+/// compiled with coverage instrumentation (-profile-generate or
+/// --enable-code-coverage in Swift).
+///
+/// Reference: https://github.com/llvm/llvm-project/blob/main/compiler-rt/include/profile/instr_prof_interface.h
+
+/// Reset all profile counters to zero.
+///
+/// Call this before running a section of code to isolate its coverage from
+/// previously executed code.
+///
+/// @note This function is only available when coverage instrumentation is
+/// enabled. Use swt_profilerRuntimeAvailable() to check availability.
+SWT_EXTERN void __llvm_profile_reset_counters(void);
```

Comment:
What happens if we reference these functions when code is _not_ compiled with coverage enabled?